lib/integrations/slack/send_on_slack_service.rb

```diff
@@ -33,10 +34,11 @@ def perform_reply
 
   def message_content
     private_indicator = message.private? ? 'private: ' : ''
+    message_text = message.content.gsub(MENTION_REGEX, '\1')
```

Comment:
Message content can be null for file only messages. Can you please check whether this code works if I send image without text?

Reply:
Good catch! It works for empty string, not for nil. Handling the case.